src/segmentation_accuracy_estimate.py

```diff
@@ -118,6 +128,7 @@ def main(args):
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("--start", type=int, default=1867)
     parser.add_argument("--end", type=int, default=2022)
+    parser.add_argument("--records_folder", type=str, default="corpus/records")
```

Comment:
don't we want to do `default = None` and fetch the Environment variable when this is not specified?